UI enhancements and SQL fixes

alondhe opened the pull request:
Fixes:
#65
#81
#79
#93

Enhancements:
#86
#80
#77

fdefalco commented on inst/sql/sql_server/concept_plausible_gender.sql:
```diff
@@ -13,18 +13,19 @@ plausibleGender = @plausibleGender
-			ON @cdmTableName.person_id = person.person_id
-		WHERE @cdmFieldName = @conceptId
-		AND person.gender_concept_id <> {@plausibleGender == 'Male'} ? {8507} : {8532} 
+		SELECT A.* 
```
Would recommend aliases that were more descriptive than A/B.

alondhe replied:
Latest commit uses more descriptive aliases

fdefalco commented on inst/sql/sql_server/field_cdm_field.sql:
```diff
@@ -7,17 +7,21 @@ Parameters used in this template:
 cdmDatabaseSchema = @cdmDatabaseSchema
 cdmTableName = @cdmTableName
 cdmFieldName = @cdmFieldName
+isPostgresql = @isPostgresql
```
Is this not handled by SqlRender? Unclear as why there is DB specific instruction.  Might just need to understand.

alondhe replied:
Misunderstanding on my part -- it's not that PG was an issue, it was the casts added in the CSV. I've removed this hack.

alondhe commented:
@fdefalco and @clairblacketer -- can you review now? I've updated the aliases to be more descriptive, removed the isPostgresql hack, and removed the cast as datetimes from the field level CSV and simply placed that into the relevant SQL templates.

alondhe commented:
Thanks @fdefalco for the review!
